broken/onnx.txt

```diff
@@ -0,0 +1,15 @@
+linux-ppc64le/onnx-12.0.0-py39he81994a_0.tar.bz2
```

Comment:
Can you also add `linux-64`, `linux-aarch64` and `osx-arm64` build here, too?

Reply:
@xhochy, thanks for the thorough review. I have added the missing archs.